src/huggingface_hub/README.md

```diff
@@ -2,57 +2,16 @@
 
 ## Download files from the Hub
 
-Three utility functions are provided to dowload files from the Hub. One
-advantage of using them is that files are cached locally, so you won't have to
+The `hf_hub_download()` function is the main function to download files from the Hub. One
```

Comment:
```suggestion
The [`hf_hub_download`] function is the main function to download files from the Hub. One
```

Reply:
if i'm not mistaken this file is not rendered in the doc (it's just displayed in GitHub) so i'll reject this proposal